project/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.sps.utilities.CommonUtils;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns the login status of the user. */
+@WebServlet("/login-status")
+public class LoginStatusServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json;");
+
+    // Get the current login status.
+    LoginStatus status;
+    if (UserServiceFactory.getUserService().isUserLoggedIn()) {
+      status = LoginStatus.getLoggedInInstance();
+    } else {
+      status = LoginStatus.getNotLoggedInInstance();
+    }
+
+    // Send the JSON as the response.
+    response.getWriter().println(CommonUtils.convertToJson(status));
```

Comment:
Is there a difference between CommonUtils.convertToJso vs gson (from the starter projects)? Should we use one over the other? (Just so we use the same method throughout the proj)

Reply:
Disregard! I see it's a function you wrote, not a built in